tools/depcheck/examples/origin_filters_extended.json

```diff
@@ -0,0 +1,184 @@
+[
```

Comment:
In the end, this should be built programatically in our openshift specific command.